### PR TITLE
status page ready

### PR DESCRIPTION
**Pull request by aniarya82**

### CODE REVIEW

**Comment by aniarya82** [comment]
screen shot 2015-05-14 at 9 37 45 pm

File: app/controllers/viewall_controller.rb [hunk 1]
```diff
@@ -11,5 +11,10 @@ def alumni
 
   def students
   	## TODO: BY hiten
```
Comment (icyflame) [inline app/controllers/viewall_controller.rb]: You can remove the TODO lines.

File: app/views/viewall/students.html.erb [hunk 2]
```diff
@@ -1,2 +1,40 @@
...
+	<tbody>
+		<% @allStudents.each do |student| %>
+		<% @searchAdd = 0 %>
+		<% @responseAdd = 0 %>
+		<tr>
+			<% allties = TieAlumniWithStudentMember.where({ :StudentMember_id => "#{student.id}" }) %>
+				<% if allties.length >= 1 %>
+					<% allties.each do |tie| %>
+					<% stat = AlumniStatus.where({ :Alumni_id => "#{tie.Alumni_id}"}) %>
+						<% if stat[0].search == "success" %>
```
Comment (icyflame) [inline app/views/viewall/students.html.erb]: Check out the enums for `search` and `response` [here](https://github.com/icyflame/networking-rails/blob/master/app/models/alumni_status.rb#L3)

File: app/views/viewall/students.html.erb [hunk 3]
```diff
@@ -1,2 +1,40 @@
...
+						<% @searchAdd = @searchAdd + 1 %>
+						<% end %>
+						<% if stat[0].response == "success" %>
```
Comment (icyflame) [inline app/views/viewall/students.html.erb]: Check the enum mentioned above and change `success` to something that is a member of the enum.

**Comment by aniarya82** [comment]
screen shot 2015-05-14 at 11 28 50 pm
How does this design looks...??

**Comment by icyflame** [comment]
@aniarya82 
Much better! If we can implement this, that would be the dream! :+1:

**Comment by aniarya82** [comment]
its is already implemented....will push now..

**Comment by icyflame** [comment]
@aniarya82 
As discussed on Slack, you may change the string in one of the enums fro `default` to `netural`.

**Comment by icyflame** [comment]
@aniarya82 
Great work! I haven't had time to review this PR yet. I will do that over the weekend.

- Meanwhile, is the route to this page `/viewall/student` ?
- And also, is the link in the navbar shown only when someone is signed in?

**Comment by aniarya82** [comment]
yup route is set...
and link is shown in the nav bar for both coordinator and student member

**Comment by icyflame** [comment]
@aniarya82 Okay!

Also, we need to change the text in the navbar. It seems pretty weird now.
I suggest:
- Change `See complete database` -> `Alumni Status`
- Change `See Student Status` -> `Student Member Status`

I think these phrases make sense, what do you say?

**Comment by aniarya82** [comment]
okay i add it to the current pr itself

**Comment by icyflame** [comment]
@aniarya82 
Yes, add it to this PR itself.

Another thing that I noticed is that the four things with labels are buttons. But they don't lead anywhere, so, it is better to remove the hyperlink.

**Comment by aniarya82** [comment]
i was thinking of adding modal to those button so that one can click to check who has responded Negative and who has responded Positive like that...
You got it na.....
Will it be good.....??????

**Comment by icyflame** [comment]
@aniarya82 
No, that is not required. Modals don't render well on phones, and anyway, in case someone wants to see that, then he/she can switch to the complete alumni DB view, and then, order that list accordingly.

I am going to add another column to the full alumni database, which has the name of the person that the alumni is assigned to. This is something that will be practically very important.

**Comment by icyflame** [comment]
@aniarya82 
I will take it from here, please do not make any changes/push to the branch anymore.

**Comment by icyflame** [comment]
@aniarya82 
Your work on this PR has been merged in the commit aed3e54.

Good going! :+1:

**Pull request closed without merging**